### ISSUE TITLE
Jekyll gets stuck on an invalid image path when using --livereload?

### ISSUE THREAD

**Issue by Gira-X** [wildlyinaccurate/jekyll-responsive-image#85]
I run this command in development: `bundle exec jekyll serve --livereload`

When the image path is incorrect once (sometimes accidentally by typing in the path to `responsive_image` slowly), the livereload feature gets stuck and does not rebuild anything anymore, even when files change:

```
.../jekyll-responsive-image/image_processor.rb:9:in `process': Invalid image path specified: assets/blub.png (SyntaxError)
```

On seeing this, I have to restart the above command by quitting out of the command manually.
Is there a setting or something I can configure to keep the command running correctly automatically?

Thank you :)

---

Jekyll version: 3.8.6 (if relevant)

**Comment by wildlyinaccurate** [comment]
Thanks for reporting this! I'm guessing the error is interfering with the live reload functionality. The "invalid image path" message should probably be logged to the terminal without actually throwing an error.

**Comment by wildlyinaccurate** [comment]
Fixed in 91a540c266392478ef31eb2416fe2ba42a2db05d and released in [1.5.3](https://rubygems.org/gems/jekyll-responsive-image/versions/1.5.3).
